core/domain/skill_domain.py

```diff
@@ -313,7 +320,8 @@ def __init__(
             self, skill_id, description, misconceptions,
             skill_contents, misconceptions_schema_version,
             skill_contents_schema_version, language_code, version,
-            next_misconception_id, created_on=None, last_updated=None):
+            next_misconception_id, created_on=None, last_updated=None,
+            superseding_skill_id=None, all_questions_merged=None):
```

Comment:
I don't think these last two fields should be optional.

Reply:
Done